api/get/run_api.sh

```diff
@@ -0,0 +1,3 @@
+export FLASK_APP=api.py
+export FLASK_DEBUG=1
+flask run
```

Comment:
Nice. I assume this is to go hand in hand with the `README` and allow anyone to use our project going forward?